README.md

```diff
@@ -113,138 +86,195 @@ You can still find the latest installers in the [24.9.2 release](https://github.
 
-    curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
-    bash Miniforge3-$(uname)-$(uname -m).sh
 
-or
+## Requirements and installers
 
-    wget "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
-    bash Miniforge3-$(uname)-$(uname -m).sh
+Latest installers with Python 3.12 `(*)` in the base environment:
 
-#### Uninstallation
+| OS      | Architecture                  | Minimum Version | File                            |
+| ------- | ----------------------------- | --------------- | ------------------------------- |
+| Linux   | x86_64 (amd64)                | glibc >= 2.17   | `Miniforge3-Linux-x86_64.sh`    |
+| Linux   | aarch64 (arm64) `(**)`        | glibc >= 2.17   | `Miniforge3-Linux-aarch64.sh`   |
+| Linux   | ppc64le (POWER8/9)            | glibc >= 2.17   | `Miniforge3-Linux-ppc64le.sh`   |
+| macOS   | x86_64                        | macOS >= 10.13  | `Miniforge3-MacOSX-x86_64.sh`   |
+| macOS   | arm64 (Apple Silicon) `(***)` | macOS >= 11.0   | `Miniforge3-MacOSX-arm64.sh`    |
+| Windows | x86_64                        | Windows >= 7    | `Miniforge3-Windows-x86_64.exe` |
 
-Uninstalling Miniforge means removing the files that were created during the installation process.
-You will typically want to remove:
+`(*)` The Python version is specific only to the base environment. Conda can create new environments with different Python versions and implementations.
 
-1. Any modifications to your shell rc files that were made by Miniforge:
+`(**)` For Raspberry PI that include a 64 bit processor, you must also use
+a 64-bit operating system such as
+[Raspberry Pi OS 64-bit](https://www.raspberrypi.com/software/operating-systems/#raspberry-pi-os-64-bit)
+or
+[Ubuntu for Raspberry PI](https://ubuntu.com/raspberry-pi).
+The versions listed as "System: 32-bit" are not compatible with the installers on this website.
 
-```bash
-# Use this first command to see what rc files will be updated
-conda init --reverse --dry-run
-# Use this next command to take action on the rc files listed above
-conda init --reverse
-# Temporarily IGNORE the shell message
-#       'For changes to take effect, close and re-open your current shell.',
-# and CLOSE THE SHELL ONLY AFTER the 3rd step below is completed.
-```
+`(***)` Apple silicon builds are experimental and haven't had testing like the other platforms.
 
-2. Remove the folder and all subfolders where the base environment for Miniforge was installed:
 
-```bash
-CONDA_BASE_ENVIRONMENT=$(conda info --base)
-echo The next command will delete all files in ${CONDA_BASE_ENVIRONMENT}
-# Warning, the rm command below is irreversible!
-# check the output of the echo command above
-# To make sure you are deleting the correct directory
-rm -rf ${CONDA_BASE_ENVIRONMENT}
+## Install
+
+On all platforms, the installation should be finalized with commands like
```

Comment:
i don't think this is true on unix. 

You should instead explain what the defaults will do, and explain why you suggestion is important.